README.md

```diff
@@ -0,0 +1,59 @@
+# gtest-parallel
+
+_This is not an official Google product._
+
+`gtest-parallel` is a script that executes [Google
+Test](https://github.com/google/googletest) binaries in parallel, providing
+good speedup on multi-core machines, especially for single-threaded tests, or
+tests that do otherwise not run at 100% CPU.
+
```

Comment:
Maybe this is easier to parse?

"... providing good speedup for single-threaded tests (on multi-core machines) and tests that do not run at 100% CPU (on single- or multi-core machines)."